ecl/dedicated_hypervisor/v1/server.py

```diff
@@ -61,6 +61,8 @@ class Server(resource2.Resource):
     availability_zone = resource2.Body('availability_zone')
     #: Connection between the server and common function gateway network is established or not.
     cfgw_connection_status = resource2.Body('cfgw_connection_status')
+    #: Get the BMDH image name
```

Comment:
```suggestion
    #: Image name of the server.
```
ここは DH (dedicated_hypervisor)のソースであり、BM (baremetal)ではありません。
ですから、少なくとも "BM"と書くのはおかしいかと。
API仕様書の説明文をそのまま持ってくるほうが無難かと思いました。

Reply:
おっしゃる通りに修正いたしました。